mindocr/data/transforms/transforms_factory.py

```diff
@@ -14,7 +14,7 @@
 
 
 # TODO: use class with __call__, to perform transformation
-def create_transforms(transform_pipeline, global_config=None):
+def create_transforms(transform_pipeline: list, global_config: dict=None):
```

Comment:
this kind of type hint is only support in python 3.9 or above, for 3.7/3.8, better to use from `typing.List[Any]` and `typing.Dict[Any, Any]`, or `from __future__ import annotations`

Reply:
> The code formatting change and the functional change is better to be separated into two commit; and once the pre-commit hook of formatting is finished, the formatting change of the entire code is better to be in one time

True. Only format part of it.

Reply:
Seems without claiming the internal parameter of the generic type, `xxx: list` is also ok